sdk/src/main/scala/com/horizen/account/abi/ABIUtil.scala

```diff
@@ -0,0 +1,29 @@
+package com.horizen.account.abi
+
+import com.horizen.account.state.ExecutionFailedException
+import com.horizen.utils.BytesUtils
+import org.web3j.crypto.Hash
+import org.web3j.utils.Numeric
+
+import java.util
+
+object ABIUtil {
```

Comment:
this class should be moved to `java` package and become java class.
Can and should be reused by developers while implementing custom msgProcessors.